skycoin/dmsg
# Feat/skywire utilities

Fixes #	

 Changes:	
- Used package `netutil` from `skywire-utilities`

How to test this PR:
1. Run `make build`
2. Check functionality

Needs:
[#skywire-utilities-3](https://github.com/skycoin/skywire-utilities/pull/3)
